astropy/visualization/units.py

```diff
@@ -6,8 +6,12 @@
 
 __doctest_skip__ = ['quantity_support']
 
+default_formt = 'latex_inline'
+braket_normal = "{}({})"
+braket_latex = r"{}$\left({}\right)$"
 
-def quantity_support(format='latex_inline'):
+
+def quantity_support(xlabel="", ylabel="", format=None):
```

Comment:
Can we move xlabel and ylabel to the end? Changing the order here will break people calling this without the keyword.

Reply:
I think keyword arguments are a good way to resolve this. There's no intrinsic reason to order the arguments in any way (except to maximize ease of use). So this is very amenable to imposing *no* order by having kwargs. To preserve backward compatibility I suggest

```python
def quantity_support(format=None, *, xlabel="", ylabel=""):
```

Also a nitpick, but technically the empty strings are mutable (see https://towardsdatascience.com/python-pitfall-mutable-default-arguments-9385e8265422 for why this is an anti-pattern). Switching to `None` might be better.